src/main/java/uk/gov/hmcts/reform/prl/controllers/CallbackController.java

```diff
@@ -237,6 +238,13 @@ public AboutToStartOrSubmitCallbackResponse generateDocumentSubmitApplication(
         caseDataUpdated.putAll(caseSummaryTab.updateTab(caseData));
         caseDataUpdated.putAll(map);
 
+        if (null != caseData.getCaseCreatedBy() && caseData.getCaseCreatedBy().equals(CaseCreatedBy.CITIZEN)) {
```

Comment:
Revert the equation. you wont need null check

Reply:
Removed